### PR TITLE
dispatch loadedmetadata after youtube player ready

### PR DESCRIPTION
**Pull request by a-b-r-o-w-n**

### CODE REVIEW

File: build/mediaelement-and-player.js [hunk 1]
```diff
@@ -1789,6 +1789,7 @@ mejs.YouTubeApi = {
 
 					mejs.YouTubeApi.createEvent(player, pluginMediaElement, 'loadstart');
 					mejs.YouTubeApi.createEvent(player, pluginMediaElement, 'canplay');
+				  mejs.YouTubeApi.createEvent(player, pluginMediaElement, 'loadedmetadata');
```
Comment (mziwisky) [inline build/mediaelement-and-player.js]: i don't know why this indentation doesn't match that of the source.  maybe your source originally had this indentation, then you built it, then fixed the source and committed without remembering to re-build it?

File: build/mediaelement-and-player.js [hunk 2]
```diff
@@ -2407,6 +2406,81 @@ window.MediaElement = mejs.MediaElement;
 
 }(mejs.i18n.locale.strings));
 /*!
+ * This is a i18n.locale language object.
```
Comment (mziwisky) [inline build/mediaelement-and-player.js:2410]: where does this whole block of I18n stuff come from?  do we have a previous commit that neglected to do the build before merging?

**Review by mziwisky — DISMISSED** [review]
you maybe need another build?  but other than that it looks fine

File: build/mediaelement-and-player.js (hunk 2 above)
Comment (a-b-r-o-w-n) [reply build/mediaelement-and-player.js:2410]: #22 changed the gruntfile a bit without doing a build it looks like. I guess this is what that included in the build. @tennety might have more info.

Comment (tennety) [reply build/mediaelement-and-player.js:2410]: Yeah, I think what @astephenb said is correct. I had missed the Gruntfile changes in #23, so I probably missed some of the build artifacts between #22 and #23. 😥

**Review by mziwisky — COMMENTED** [review]
looks good.  looks like you probably did a `push --force` for the update?  i think going forward we should  just push up new commits in the branch, leaving the old behind for a cleaner history, and then do "Squash and Merge" when it's approved.  gotta get our brains out of gerrit-mode when we're in github land.

**Review by mziwisky — APPROVED** [review]

**Pull request merged**